ark/phenotyping/cell_cluster_utils.py

```diff
@@ -426,24 +421,24 @@ def train_cell_som(fovs, base_dir, pixel_data_dir, cell_table_path, cell_som_clu
     """
 
     # define the data paths
-    cluster_counts_size_norm_path = os.path.join(base_dir, cluster_counts_size_norm_name)
+    cell_som_input_data_path = os.path.join(base_dir, cell_som_input_data_name)
     som_weights_path = os.path.join(base_dir, som_weights_name)
 
-    # check the cell table path exists
-    io_utils.validate_paths([cell_table_path])
+    # check the cell SOM inputs cell table path exists
+    io_utils.validate_paths([cell_som_input_data_path, cell_table_path])
 
-    # load the normalized pixel count expression data for verification
-    cluster_counts_size_norm_data = feather.read_dataframe(cluster_counts_size_norm_path)
+    # load the cell SOM input data for verification
+    cell_som_input_data = feather.read_dataframe(cell_som_input_data_path)
 
     # verify the cell_som_cluster_cols columns provided are valid
     misc_utils.verify_in_list(
         provided_cluster_cols=cell_som_cluster_cols,
-        som_input_cluster_cols=cluster_counts_size_norm_data.columns.values
+        som_input_cluster_cols=cell_som_input_data.columns.values
     )
 
     # define the cell SOM cluster object
     cell_pysom = cluster_helpers.CellSOMCluster(
```

Comment:
Question for @ngreenwald. In the constructor for `CellSOMCluster`, `normalize_data` is called - which divides each input feature by its 99.9th percentile. Is that something that we always want to do?

Reply:
I'm not sure. We'll need to wait and see how the probabilities behave for clustering. It's probably a good idea, but we may decide it makes thing worse and want to change this a user-defined step

Reply:
I feel like in my head, 99.9 normalization is meant to help with markers that are always waaaay brighter (or dimmer) than other markers. If the input features are probabilities, they should all be in the same range anyways (like a marker shouldn't always have suuuuper low probabilities - if so, that's probably just a crappy marker and should've been thrown out anyways). I think it might make more intuitive sense (at least to me) to first try it without normalization, and then add it back in later if the clustering looks bad. But anyways, I agree that it could probably wait until we see how the data is behaving.

Reply:
Let's leave this on the back burner then. I'll open an issue for this if and when the time comes.